components/Table/PlayerMatchesTable.container.jsx

```diff
@@ -29,7 +29,7 @@ class RequestLayer extends React.Component {
     this.props.getPlayerMatches(this.props.playerId, this.props.numMatches);
   }
 
-  componentWillReceiveProps(nextProps) {
+  componentWillUpdate(nextProps) {
```

Comment:
I think this is a preferred lifecycle method since I don't think it's called when the component is first loaded